dolphinscheduler-api/src/test/java/org/apache/dolphinscheduler/api/service/TaskDefinitionServiceImplTest.java

```diff
@@ -50,48 +51,29 @@
 @RunWith(MockitoJUnitRunner.class)
 public class TaskDefinitionServiceImplTest {
 
-    String taskDefinitionJson = "{\n"
```

Comment:
This variable is referenced by many test cases. Does the modification affect other test cases?